examples/util/BaseRunner.js

```diff
@@ -103,283 +132,168 @@ class BaseRunner {
           }
         }
       };
-      if (handle != null) {
-        oReq.onprogress = handle;
+      if (handler != null) {
+        oReq.onprogress = handler;
       }
       oReq.send();
     });
   };
 
-  _getRawModel = async (url) => {
-    let status = 'ERROR';
-    let rawModel = null;
-
-    if (url !== undefined) {
-      const arrayBuffer = await this._loadURL(url, this._progressHandler, true);
-      const bytes = new Uint8Array(arrayBuffer);
-      switch (url.split('.').pop()) {
-        case 'tflite':
-          const flatBuffer = new flatbuffers.ByteBuffer(bytes);
-          rawModel = tflite.Model.getRootAsModel(flatBuffer);
-          rawModel._rawFormat = 'TFLITE';
-          status = 'SUCCESS'
-          printTfLiteModel(rawModel);
-          break;
-        case 'onnx':
-          const err = onnx.ModelProto.verify(bytes);
-          if (err) {
-            throw new Error(`The model file ${url} is invalid, ${err}`);
-          }
-          rawModel = onnx.ModelProto.decode(bytes);
-          rawModel._rawFormat = 'ONNX';
-          status = 'SUCCESS'
-          printOnnxModel(rawModel);
-          break;
-        case 'bin':
-          const networkFile = url.replace(/bin$/, 'xml');
-          const networkText = await this._loadURL(networkFile);
-          const weightsBuffer = bytes.buffer;
-          rawModel = new OpenVINOModel(networkText, weightsBuffer);
-          rawModel._rawFormat = 'OPENVINO';
-          status = 'SUCCESS';
-          break;
-        default:
-          throw new Error(`Unrecognized model format, support TFLite | ONNX | OpenVINO model`);
-      }
-    } else {
-      throw new Error(`There's none model file info, please check config info of modelZoo.`);
-    }
-
-    this._setRawModel(rawModel);
-    this._setLoadedFlag(true);
-    return status;
-  };
-
-  _getOtherResources = async () => {
-    // Override by inherited if needed, likes load labels file
-  };
-
-  _getModelResources = async () => {
-    await this._getRawModel(this._currentModelInfo.modelFile);
-    await this._getOtherResources();
-  };
-
+  /** @override */
+  _doInitialization = (modelInfo) => {};
```

Comment:
/** @override */ should be used in sub-class. In base class, please put the method document here by following JS Doc style.